src/components/prompts/PredictionsWarning.js

```diff
@@ -50,65 +52,63 @@ class TermsAndConditions extends Component {
 
     return (
       <Modal
-        id={ `WarningModal` }
-        mountNode = { document.getElementById('App') }
-        size='large'
-        open={ !termsAccepted }
-        closeOnDimmerClick={ false }
-        closeOnEscape={ false }>
-        <Modal.Header> 
-          { translations.i_header }
-        </Modal.Header>
+        id                 = { `WarningModal` }
+        mountNode          = { document.getElementById(`App`) }
+        size               = { `large` }
+        open               = { !termsAccepted }
+        closeOnDimmerClick = { false }
+        closeOnEscape      = { false }>
+
+        <Modal.Header>{ translations.i_header }</Modal.Header>
+
         <Modal.Content scrolling>
 
           { translations.i_warning } 
 
           <h4>{ translations.i_formInstructions }</h4>
 
           <div
-            className="radio-yes-no"
-            key= { `ReqCkBx1` }>
+            className = { `radio-yes-no` }
+            key       = { `ReqCkBx1` }>
             <Form.Field>
               <Checkbox
                 checked = { this.state.Checkbox1 }
                 name    = { `checkbox1` }
-                onClick = { () => {return this.handleChange('checkbox1');} } />
-            </Form.Field>
-            <Form.Field>
-              { translations.i_checkboxLabel1 }
+                onClick = { () => { return this.handleChange(`checkbox1`); } } />
```

Comment:
As long as we're cleaning up code, we might as well make these handlers member functions instead of anonymous functions (see #974)